tests/components/google_assistant/test_google_assistant.py

```diff
@@ -209,6 +227,7 @@ def test_execute_request(hass_fixture, assistant_client):
     body = yield from result.json()
     assert body.get('requestId') == reqid
     commands = body['payload']['commands']
-    assert len(commands) == 2
+    assert len(commands) == 3
     ceiling = hass_fixture.states.get('light.ceiling_lights')
     assert ceiling.state == 'off'
+    assert hass_fixture.states.get('switch.decorative_lights').state == 'off'
```

Comment:
How does this test that it was exposed as a light?

Reply:
This test doesn't. Just that is could be turned of, exposed as a light, but handled as a switch.

The type is tested in the sync test https://github.com/perosb/home-assistant/blob/4f569654873b3f3fdece4e108b7200443f801200/tests/components/google_assistant/test_google_assistant.py#L153